src/client/ui/selector-inspector-panel/index.js

```diff
@@ -0,0 +1,38 @@
+import uiRoot from '../ui-root';
+
+import { createElementFromDescriptor } from './utils/create-element-from-descriptor';
+import { setStyles } from './utils/set-styles';
+
+import { panel } from './descriptors';
+import { PickButton } from './pick-button';
+import { SelectorInputContainer } from './selector-input-container';
+import { CopyButton } from './copy-button';
+import { elementPicker } from './element-picker';
+
+export default class SelectorInspectorPanel {
+    element;
```

Comment:
I do like this approach, when we mix this type of properties definition and constructor.
@miherlosev what do you think?

Reply:
@felis2803 uses the declarative react like approach. We usually the more 'plain` approach. I think that in this case we leave it as is since the code is regular.